docs/AnnotatedDataset.md

```diff
@@ -1,37 +1,44 @@
-# Annotated Dataset 
+# Annotated dataset 
```

Comment:
I've tried to make it more clear who is doing what in this file, for example are we talking about the authors of the paper or the Synth Det team. This is to align with this guideline in the docs style guide: https://unity-docs.gitbook.io/style-guide/getting-started/writing-tips#use-active-voice 
Let me know if I haven't attributed things correctly, and also if it's OK to refer to the Synth Det team, as I've done a few times.